<!doctype html><html lang=en-us><head><meta name=generator content="Hugo 0.108.0"><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=theme-color content="dark"><title>Etowah Adams</title><meta property="og:site_name" content="Etowah Adams"><meta property="og:title" content="Etowah Adams"><meta itemprop=name content="Etowah Adams"><meta name=twitter:title content="Etowah Adams"><meta name=application-name content="Etowah Adams"><meta name=twitter:card content="summary"><meta name=description content><meta name=twitter:description content><meta itemprop=description content><meta property="og:description" content><link rel="shortcut icon" type=image/x-icon href=/favicon.ico><link rel=stylesheet href=/sass/main.min.ab99ff095f832511e24ffb2fba2b51ad473b2f7e9301d674eba2c6c3a6e8bd81.css></head><script>(function(){const e="ThemeColorScheme";localStorage.setItem(e,"light")})()</script><script>(function(){const t="ThemeColorScheme",e=localStorage.getItem(t),n=window.matchMedia("(prefers-color-scheme: dark)").matches===!0;e=="dark"||e==="auto"&&n?document.documentElement.dataset.userColorScheme="dark":document.documentElement.dataset.userColorScheme="light"})()</script><body class=dark><nav class=navbar><div class=container><div class=flex><div><a class=brand href=/>Etowah Adams</a></div><div class=flex><a href=/articles/>Articles</a></div></div></div></nav><main><div class=container><section class=my><div class=content><p>Hello! I&rsquo;m Etowah, a second year PhD student in Systems Biology at Columbia studying machine learning and biology. I am advised by Prof. <a href=https://www.aqlab.io/>Mohammed AlQuraishi</a>. In my research I seek to build increasingly complete (re: multi-modal, temporal, molecular) representations of biology to understand and engineer biology.</p><figure><a data-fancybox href=array.png data-caption="No, not a Pollock painting. It&rsquo;s a DNA sequencing flow cell in action."><img src=array.png data-src=array.png class=lazyload alt></a><figcaption style=text-align:left>No, not a Pollock painting. It&rsquo;s a DNA sequencing flow cell in action.</figcaption></figure><p>In my undergrad at Yale, I developed CRISPR-based gene editing tools with Prof. <a href="https://scholar.google.com/citations?user=OCyNKlYAAAAJ&hl=en">Farren Isaacs</a>. Subsequently, I worked at Harvard Medical School as a research software engineer with Prof. <a href=https://hidivelab.org/team/members/nils-gehlenborg/>Nils Gehlenborg</a> on <a href=http://gosling-lang.org/>tools</a> for genomics data. I also developed software at <a href=https://www.ginkgobioworks.com/>Ginkgo Bioworks</a> as an intern. I&rsquo;m originally from southern Applachia where I grew up on a small farm.</p><p>I am happy to chat about research or anything else! Feel free to reach out. A few things I love discussing: self-supervised learning, mechanistic interpretability, protein structure prediction, amortized sampling, cloud labs, optimal experimental design, expansion microscopy, CryoET, WebGL, and banjo.</p><p><a href=https://x.com/etowah0>Twitter</a> | <a href="https://scholar.google.com/citations?user=brjaKFEAAAAJ&hl=en">Google Scholar</a> | <a href=https://github.com/etowahadams>GitHub</a> | <a href=https://www.linkedin.com/in/etowah/>LinkedIn</a> | <a href=mailto:etowahadams@gmail.com>Email</a></p><div class=papers><h2 class=papers-year-heading>2025</h2><article class=paper><img src=https://etowahadams.github.io/papers/interprot.png alt="From Mechanistic Interpretability to Mechanistic Biology: Training, Evaluating, and Interpreting Sparse Autoencoders on Protein Language Models thumbnail" class=paper-thumb loading=lazy><div class=paper-body><h5 class=paper-title>From Mechanistic Interpretability to Mechanistic Biology: Training, Evaluating, and Interpreting Sparse Autoencoders on Protein Language Models</h5><div class=paper-venue>International Conference on Machine Learning, 2025 (Spotlight)</div><div class=paper-authors><strong>Etowah Adams*</strong>, Liam Bai*, Minji Lee, Yiyang Yu, Mohammed AlQuraishi</div><div class=paper-comment><em>First paper of my PhD! Using sparse autoencoders, we try to figure out what protein langauge models have learned about proteins.</em></div><div class=paper-links><a class=paper-link href="https://openreview.net/forum?id=zdOGBRQEbz" target=_blank rel=noopener>Paper</a><a class=paper-link href=https://interprot.com target=_blank rel=noopener>Project</a><a class=paper-link href=https://github.com/etowahadams/interprot target=_blank rel=noopener>Code</a><a class=paper-link href=https://transformer-circuits.pub/2025/july-update/index.html#bio target=_blank rel=noopener>Anthropic Coverage</a><a class=paper-link href=https://x.com/etowah0/status/1888977417127055413 target=_blank rel=noopener>Tweet Thread</a></div></div></article><article class=paper><img src=https://etowahadams.github.io/papers/evo2.png alt="Genome modeling and design across all domains of life with Evo 2 thumbnail" class=paper-thumb loading=lazy><div class=paper-body><h5 class=paper-title>Genome modeling and design across all domains of life with Evo 2</h5><div class=paper-venue>Preprint</div><div class=paper-authors>Garyk Brixi*, Matthew G. Durrant*, Jerome Ku*, Michael Poli*, Greg Brockman, ..., <strong>Etowah Adams</strong>, ..., Dave P. Burke, Hani Goodarzi, Patrick D. Hsu, Brian L. Hie</div><div class=paper-comment><em>I led engineering of Evo Designer. Huge thanks to Arc Insitute and Patrick Collison for the opportunity to contribute!</em></div><div class=paper-links><a class=paper-link href=https://arcinstitute.org/manuscripts/Evo2 target=_blank rel=noopener>Paper</a><a class=paper-link href=https://arcinstitute.org/tools/evo target=_blank rel=noopener>Project</a><a class=paper-link href=https://www.nature.com/articles/d41586-025-00531-3 target=_blank rel=noopener>Media</a></div></div></article><article class=paper><img src=https://etowahadams.github.io/papers/anabel.png alt="Precision multiplexed base editing in human cells using Cas12a-derived base editors thumbnail" class=paper-thumb loading=lazy><div class=paper-body><h5 class=paper-title>Precision multiplexed base editing in human cells using Cas12a-derived base editors</h5><div class=paper-venue>Nature Communications</div><div class=paper-authors>Anabel Schweitzer, <strong>Etowah Adams</strong>, Michael Nguyen, Monkol Lek, Farren J. Isaacs</div><div class=paper-comment><em>Paper from my undergrad. I developed analysis tools for base editing outcome sequencing data.</em></div><div class=paper-links><a class=paper-link href=https://www.nature.com/articles/s41467-025-59653-x target=_blank rel=noopener>Paper</a><a class=paper-link href=https://github.com/etowahadams/EditRBatch target=_blank rel=noopener>Code</a></div></div></article></div><div class=bio-social></div></div></section></div><div class=container><section><h2>Latest Articles
<a class=section-button href=/articles/>View all</a></h2><div class=posts><div class=post><a href=/post/2024-07-17-cloning/><div class=post-row><time>Jul 16</time><h3>Molecular cloning, or why some biologists become superstitious</h3></div></a></div><div class=post><a href=/post/2020-01-09-maple-red-leaf/><div class=post-row><time>Jan 09</time><h3>Leaf color change in 10,950 iNaturalist observations</h3></div></a></div></div></section></div></main><footer class="footer flex"><section class=container><nav class=footer-links><a href=/index.xml>RSS</a></nav></section><script defer src=/ts/features.dd8e5b18a6dda91367e396fd8dc9b9f83a1993065f709a53e056e0f2e9096ea1.js data-enable-footnotes=true></script></footer></body></html>